docs/spark/spark-procedures.md

```diff
@@ -198,7 +198,7 @@ the `expire_snapshots` procedure will never remove files which are still require
 | Argument Name | Required? | Type | Description |
 |---------------|-----------|------|-------------|
 | `table`       | ✔️  | string | Name of the table to update |
-| `older_than`  | ️   | timestamp | Timestamp before which snapshots will be removed (Default: 5 days ago) |
+| `older_than`  | ️   | timestamp | Timestamp before which snapshots will be removed (Default: 5 days ago). Considered upto millisecond precision. |
```

Comment:
"up to" - although in general i'm not sure this clarification is needed, do you think this is a confusion among users?

Reply:
I think people assume it is microsecond precision as spark is microsecond precision, but code we are losing the precision (lossy conversion to milli), so I think better to clarify in the docs.

Reply:
I'm just not sure how often folks would expect their microsecond values to matter in snapshot expiration.

Reply:
ok, as snapshots itself is having creation time in milliseconds, we cannot expect to have microsecond precision. 
I will remove it.